Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -151,12 +151,19 @@ public enum AssertionShowMode { None, Implicit, All }
 java - Compile to Java.
 py - Compile to Python.
 cpp - Compile to C++.
+rs - Compile to Rust.
```

Comment:
Rust is not a fully supported target at this point, so I'm not sure this line should be here.

Reply:
Perhaps adding `[EXPERIMENTAL] `? Removing it from the list seems improper since it is technically an available option. I'll remove it still if you'd like, but just wanted to offer that as point.

Reply:
OK !